### DISCUSSION TITLE
Toggle Views

### DISCUSSION THREAD

**Discussion by cemalgnlts** [CodinGame/monaco-vscode-api#302] [Q&A] [answered]
Hi,

I don't want to show Explorer at startup, I created an extension for it and I want to show it by default, is there any way to do that?

**Comment by CGNonofr** [comment, marked as answer]
Not currently, it needs to be split

The only hack I can imagine is to initialize the storage service with a value that moves the explorer to another part or hide it

**Reply by cemalgnlts** [reply]
Thanks for the idea.

**Reply by cemalgnlts** [reply]
I discovered that I can open my own view with `workbench.view.extension.[viewName]`. Running this command after init everything works for now.

**Comment by CGNonofr** [comment]
But out of curiosity, why do you need tour own explorer?

**Reply by cemalgnlts** [reply]
What I'm trying to do: Scanning the project folder and listing the projects. I already do this and when the project is selected, I open that folder. The problem is that Explorer opens by default and I want to open the extension I made instead, so that my project extension will be opened first, the project will be selected and then I will open the Explorer view.
